decentraland/unity-explorer
# chore: update default lod value to 5

## What does this PR change?

Fixes #980 

Had to add the set of the dafult value in `PersistentSetting` since I had an outdated value in PlayerPrefs, and it was returning me that one instead of the default value I wanted. 

## How to test the changes?



1. Launch the explorer
2. Enable LOD debugging
3. Check that scenes are green up to 5 radius

![image](https://github.com/decentraland/unity-explorer/assets/1999557/5a5d74df-14f3-41cf-b1ff-97bf12b16fdb)


## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
